requirements_dev.txt

```diff
@@ -8,6 +8,6 @@ coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
 
-pytest==6.3.1
+pytest==7.3.1
```

Comment:
I've just pushed a change to master that updates all dev requirements to the latest versions, which support 3.7. Feel free to rebase to pick up this commit